server_test.go

```diff
@@ -65,3 +72,87 @@ func TestLoad(t *testing.T) {
 		<-results
 	}
 }
+
+func TestIndex(t *testing.T) {
```

Comment:
Tests functions are named according to the functions they test. This should be named `TestHandleIndex` for example.

Reply:
Fixed